Source/CarthageKit/Project.swift

```diff
@@ -226,23 +226,25 @@ public final class Project { // swiftlint:disable:this type_body_length
 			.startOnQueue(cloneOrFetchQueue)
 	}
 
-	func downloadBinaryFrameworkDefinition(url: URL) -> SignalProducer<BinaryProject, CarthageError> {
+	// Dane - what is this value used for?
```

Comment:
are you trying to ask something or did you just forget the comments in 😄  ?

Reply:
totally forgot to clean up some notes I was following around code